priv/repo/migrations/0500_add_users_table.exs

```diff
@@ -5,8 +5,6 @@ defmodule Radiator.Repo.Migrations.AddUsersTable do
     create table(:auth_users) do
       add :name, :string
       add :email, :string
-      add :display_name, :string
```

Comment:
I did think the display_name (although not yet fully utilized) is something worth having in the Auth user table. As I wanted the `name` to be fixed per instance, having a way to specify how it is displayed seems nice here. Delegating that to the Person that is used for metadata does not seem right to me.

E.g. when refering to the logged in user, imho these values here should be used and shown, not something that might be changed by someone else than me. (In that case, the avatar also might need to bubble up, however, I did not want to make binary handling a dependency here)